content/06-cardano-components/10-cardano-serialization-lib.mdx

```diff
@@ -0,0 +1,39 @@
+---
+title: Cardano serialization library
+metaTitle: Cardano serialization library
+---
+import { OpenInNew } from 'styled-icons/material/OpenInNew'
+import { User } from 'styled-icons/boxicons-regular/User'
+
+This is a library for serialization and deserialization of data structures used in Cardano’s Haskell implementation of Shelley along with useful utility functions.
+
+## Using the library
+
+Due to the portability of Rust, you can bind to the native Rust library from any common programming language (including C and WebAssembly). This library generates both [Typescript](https://www.typescriptlang.org/) and [Flow](https://flow.org/) type definitions, and you can check out possible options by just looking at Flow types, which you can find [here](https://github.com/Emurgo/cardano-serialization-lib/blob/master/rust/pkg/cardano_serialization_lib.js.flow). You can also explore the [example folder](https://github.com/Emurgo/cardano-serialization-lib/tree/master/example) to see how to use this library from Typescript and experiment with it.
```

Comment:
embed the link on the text for Flow types, rather than saying 'you can find here'